modules/apps/transfer/types/ack.go

```diff
@@ -0,0 +1,27 @@
+package types
+
+import (
+	"fmt"
+
+	channeltypes "github.com/cosmos/ibc-go/v8/modules/core/04-channel/types"
+)
+
+// NewForwardErrorAcknowledgement returns a new error acknowledgement with path forwarding information.
+func NewForwardErrorAcknowledgement(packet channeltypes.Packet, ack channeltypes.Acknowledgement) channeltypes.Acknowledgement {
```

Comment:
Would it make sense to move these constructor functions to the internal package? Or do we expect app/middleware developers to make use of them as well?

Reply:
Applied suggestions in https://github.com/cosmos/ibc-go/pull/6731/commits/0b51eba66343366b2982fddddfea890d4b3acf68

Reply:
Thank you! :)